code/__HELPERS/unsorted.dm

```diff
@@ -1285,7 +1285,7 @@ proc/pick_closest_path(value, list/matches = get_fancy_list_of_atom_types())
 
 #define RANDOM_COLOUR (rgb(rand(0,255),rand(0,255),rand(0,255)))
 
-#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, /proc/qdel, item), time, TIMER_STOPPABLE)
+#define QDEL_IN(item, time) addtimer(CALLBACK(GLOBAL_PROC, .proc/qdel, item), time, TIMER_STOPPABLE)
```

Comment:
It doesn't actually matter, the only reason i suggested using `.` for global procs was to keep it more inline with object procs.

`.` is just a short cut here for a parent lookup, the full path of the proc will always work, its just a pain for longer path types where you don't know what type added the proc and you don't want to have to go look that up.